python/pdo/contract/state.py

```diff
@@ -231,14 +231,18 @@ def get_current_state_hash(ledger_config, contract_id) :
         :param ledger_config dictionary: ledger configuration that must contain 'LedgerURL'
         :param contract_id str: contract identifier
         """
-        client = sawtooth.helpers.pdo_connect.PdoRegistryHelper(ledger_config['LedgerURL'])
+        registry_helper = create_submitter(ledger_config)
 
         try :
             # what is returned when the contract has been created but
             # state has not been initialized? need to detect this and
             # return some kind of "not found" exception
-            contract_state_info = client.get_ccl_info_dict(contract_id)
-            current_state_hash = contract_state_info['current_state']['state_hash']
+            state_info = registry_helper.get_current_state_hash(contract_id)
+            if state_info['is_active']:
```

Comment:
here you either return or throw an exception, so you never reach the last line of the function below (the other return statement). Could you modify it to something like `if error; then exception; endif; return`?

Reply:
It actually looks like `current_state_hash` is no longer used, so the last return statement could be removed.

Reply:
thank you both, removed the last return line.